andela/activo-web
# #157507779 Configure Loaders

#### What does this PR do?
Configure loaders for webpack

#### Description of task to be completed?
- install file-loader for bundling file types
- install css-loader, style-loader and sass-loader for bundling styles
- install awesome-typescript-loader for bundling files with .ts, and .tsx extensions
- create the pages directory to house AuthPage directory and files
- create AuthPage.scss file to test loaders' functionality

#### How should this be manually tested?
After cloning the repo, change directory into it and run `yarn install` to install the dependencies
- Run `yarn start:dev` to start the `webpack-dev-server` in development mode
- Browser should open to `localhost:8080`

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
- https://www.pivotaltracker.com/story/show/157507779
